### PR TITLE
[TASK] Add logging to see the signing process in action. 

### PR DESCRIPTION
**Pull request by dharjeezy**
Add logging to see the signing process in action according to #89

### CODE REVIEW

File: dkg-gadget/src/worker.rs [hunk 1]
```diff
@@ -636,13 +636,16 @@ where
 		let mut keys_to_gossip = Vec::new();
 		let mut rounds_send_result = vec![];
 		let mut next_rounds_send_result = vec![];
+		let mut current_authority_id= None;
```
Comment (appcypher) [inline dkg-gadget/src/worker.rs:639]: Can you format the code?

File: dkg-gadget/src/worker.rs [hunk 2]
```diff
@@ -671,13 +679,18 @@ where
...
 					let is_keygen_finished = next_rounds.is_keygen_finished();
 					if is_keygen_finished {
 						debug!(target: "dkg", "🕸️  Queued DKGs keygen has completed");
 						self.queued_keygen_in_progress = false;
 						let pub_key = next_rounds.get_public_key().unwrap().to_bytes(true).to_vec();
+						debug!(
+								target: "dkg-signing", "Next Authority({}) DKG PublicKey Generated (Compressed): 0x{}",
+								id.clone(),
+								hex::encode(pub_key.clone())
+						);
```
Comment (appcypher) [inline dkg-gadget/src/worker.rs:693]: Same here

File: dkg-gadget/src/worker.rs [hunk 3]
```diff
@@ -636,13 +636,16 @@ where
...
 		if let Some(mut rounds) = self.rounds.take() {
+
 			if let Some(id) =
 				self.key_store.authority_id(self.current_validator_set.authorities.as_slice())
 			{
-				debug!(target: "dkg", "🕸️  Local authority id: {:?}", id.clone());
-				rounds_send_result = send_messages(&mut rounds, id, self.get_latest_block_number());
+				debug!(target: "dkg-signing", "🕸️  Local authority id: {:?}", id.clone());
+				rounds_send_result = send_messages(&mut rounds, id.clone(), self.get_latest_block_number());
+				current_authority_id = Some(id.clone());
```
Comment (appcypher) [inline dkg-gadget/src/worker.rs:648]: Should be able to clone `id` once here

**Review by appcypher — CHANGES_REQUESTED** [review]

**Review by appcypher — APPROVED** [review]

**Review by drewstone — COMMENTED** [review]
We're already logging the pubkeys in the pallets. We can close this.

**Pull request closed without merging**